datasets/us/mi/med_exclusions/crawler.py

```diff
@@ -0,0 +1,88 @@
+from typing import Dict
+from rigour.mime.types import XLSX
+from openpyxl import load_workbook
+
+from zavod import Context, helpers as h
+
+
+def crawl_item(row: Dict[str, str], context: Context):
+
+    if not row.get("entity_name"):
+        return
+
+    entity_name = row.pop("entity_name")
+    entity = context.make("LegalEntity")
```

Comment:
Looks safe to be more specific

```suggestion
    entity = context.make("Company")
```